contrib/starship/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "starship"
+pkgver = "1.17.1"
+pkgrel = 0
+build_style = "cargo"
+make_build_args = [
+    "--no-default-features",
+    "--features=battery,notify,gix-faster",
```

Comment:
with gix-faster zlib-ng is not even used, so you can remove it from here

Reply:
similarly, cmake can be removed from hostmakedeps, and dbus-devel/libgit2-devel/openssl-devel can also be removed (they seem unused and not linked to?)

Reply:
the default is gix-max-perf which fails (afaict due to a version mismatch) perhaps we it could be kept and re-enbaled on the next point release assuming it's fixed by then?

Reply:
nah

Reply:
ok